### ISSUE TITLE
After the pac3 update Pointshop2 does not display default hats.

### ISSUE THREAD

**Issue by Comminux** [Kamshak/Pointshop2#159]
Description:

Аfter updating pac3 to last version (All pac3 vesions after March 5 2021: https://github.com/CapsAdmin/pac3/commit/6878cb703bf3327619c4066be8ada67d5060b7ab ), Pointshop2 does not display default hats.

![2021-04-14-205525_scrot](https://user-images.githubusercontent.com/14937307/114756929-eb4dce00-9d63-11eb-8de4-1314a0bf0269.png)

Additional info:

Pointshop2 git version: https://github.com/Kamshak/Pointshop2/commit/ebff5036bd9f3a90ca419ece9697e812266bba07

Garry's Mod version:
```
Protocol version 24
Exe version 2020.10.14 (garrysmod)
Exe build: 10:54:09 Apr 12 2021 (8140) (4000)
GMod version 2021.04.12, branch: x86-64, multicore: 0
```

Steps to reproduce:

cd addons/pac3
git checkout 6878cb703bf3327619c4066be8ada67d5060b7ab

**Comment by askrealcookie** [comment]
Yes that's known issue, but it doesn't depend on PS2 really, random PAC3 update broke this, currently the only way to solve it is to downgrade pac3 and regenerate icons/delete spawnicons folder, depending on versions etc. If someone needs help with that or something else just come to Discord https://discord.gg/N9DmwwX
